spec.md

```diff
@@ -2250,7 +2386,7 @@ f("n")                                          # 2
 ### Index expressions
```

Comment:
Above: Update "Membership tests" to specify the `in` / `not in` operators for bytes. `65 in b"A"` returns True in Python.

Reply:
It's in the phase 2 TODO list under bytes.